pilot/util/default.cfg

```diff
@@ -94,8 +94,8 @@ timing_file: pilot_timing.json
 
 [Information]
 
-# Path to cache
-cache_dir: /lustre/atlas/proj-shared/csc108/debug/atlas/HPC_pilot_test/queue_cache
+# Path to local cache
+cache_dir:  #/lustre/atlas/proj-shared/csc108/debug/atlas/HPC_pilot_test/queue_cache #for Titan
```

Comment:
I fear that such comment will not work, 
you need to comment whole line and put empty value for cache_dir below

Reply:
Right. And updating the config parser is on the longer term todo list, so this case should be allowed.